Mendix 11 Beta 1

MarkvanMents opened the pull request:
New documentation for Mendix 11 Beta 1

ConnorLand commented on content/en/docs/appstore/use-content/platform-supported-content/modules/SAML/_index.md:
```diff
@@ -153,6 +153,8 @@ There are different versions of the SAML module, depending on which version of M
 
         Mendix 10 does not support the 2.0 version of the Atlas UI, so you should never use the even-numbered (Atlas 2.0-compatible) SAML patch releases with your Mendix 10 app.
 
+    {{% todo %}} Do we need to add anything here for Mendix 11? {{% /todo %}}
```
Currently checking with styling experts, but fairly certain this can stand as is.

ConnorLand commented on content/en/docs/appstore/use-content/platform-supported-content/modules/SAML/_index.md:
```diff
@@ -175,6 +177,7 @@ By default, the SAML module will be installed as the **SAML20** module in your a
             Only use this setting if you are using SAML on a hybrid mobile app. Note that this functionality also requires mobile authentication tokens to be enabled in your IdP Configuration as well as changes to the hybrid app package as described in [How To Implement SSO on a Hybrid App with Mendix and SAML](/howto8/mobile/implement-sso-on-a-hybrid-app-with-mendix-and-saml/).
 
         {{% alert color="warning" %}}Hybrid mobile apps are not available in Mendix 10.{{% /alert %}}
+        {{% todo %}} How about in Mendix 11? {{% /todo %}}
```
They are also not supported in Mx11. Will update this doc in the best way (we want to avoid mentioning Hybrid almost at all as it's unsupported).

ConnorLand replied:
Resolved in my upcoming PR.

ConnorLand commented on content/en/docs/refguide/general/mx-command-line-tool/app.md:
```diff
@@ -13,7 +13,7 @@ Typically, these commands require a path to the *.mpr* file as a parameter.
 ## mx create-project Command {#create-project}
 
-The `mx create-project` command creates a new app in Studio Pro. The app version depends on the version the tool was bundled with. For example, if you are using the mx tool for Studio Pro 10.0.0, `mx create project` will create a new app in that version. 
+The `mx create-project` command creates a new app in Studio Pro. The app version depends on the version the tool was bundled with. For example, if you are using the mx tool for Studio Pro 11.0.0, `mx create project` will create a new app in that version. 
```
"Do we need to add anything here regarding the new session management using authentication tokens?" - Y

@ConnorLand

ConnorLand replied:
Nothing else besides number change needed here.

ConnorLand commented on content/en/docs/refguide/mobile/introduction-to-mobile-technologies/progressive-web-app.md:
```diff
@@ -8,16 +8,6 @@ aliases:
 
 ## Introduction
 
-{{% alert color="info" %}}
```
"I don't think this info alert is relevant to Mendix 11 anymore, but please confirm." - Y

I will see if this will be true for any Mx11 minor/patch version and then amend/prepare the message for Mx11.

ConnorLand commented on content/en/docs/refguide/mobile/using-mobile-capabilities/native-language-change.md:
```diff
@@ -18,7 +18,7 @@ While this guide gives your end-user the ability to switch their app's language,
 Before starting this guide, make sure you have completed the following prerequisites:
 
-* Install Mendix Studio Pro version [9.14.0](/releasenotes/studio-pro/9.14/) or above
+* Install Mendix Studio Pro version [9.14.0](/releasenotes/studio-pro/9.14/) or above 
```
I will change this to 11.0

ConnorLand replied:
Changed in my PR

ConnorLand commented on content/en/docs/refguide/modeling/dev-best-practices.md:
```diff
@@ -311,9 +311,9 @@ Documents used to support integration should have the prefixes listed below.
+The [Deep Link](/appstore/modules/deep-link/) module is deprecated. It is replaced by [page URLs](/refguide/page-properties/#url) and [microflow URLs](/refguide/microflow/#url). For instructions on migrating to page and microflow URLs, see the [Migrating to Page and Microflow URLs](/appstore/modules/deep-link/#migrate-page-micro) section in *Deep Link*.
 
-We will continue to actively support this module for Mendix 9.
+We will continue to actively support this module for Studio Pro 9.
```
https://github.com/mendix/Mx11-docs/pull/1/files#r2018741038

I will verify w widget devs if this should be SP 9 or Mendix 9.

ConnorLand commented on content/en/docs/refguide/runtime/mendix-client/react.md:
```diff
@@ -71,7 +67,7 @@ JavaScript actions must not use any of the following client APIs:
 These APIs are related to Dojo and are no longer needed. For opening pages, please use the nanoflow [Show Page](/refguide/show-page/) action instead.
 
-Fore more information on React and APIs in Mendix, see our [Mendix 10 React Client API](https://apidocs.rnd.mendix.com/10/client-react/index.html) documentation.
+Fore more information on React and APIs in Mendix, see our [Mendix 10 React Client API](https://apidocs.rnd.mendix.com/10/client-react/index.html) documentation {{% todo %}} Do we need a Mendix 11 version here?{{% /todo %}}.
```
I will check with the team, replace this link with the Mx11 version, and then fix the "Fore" typo in both versions.

ConnorLand commented:
As I resolve all my comments, I'll need to ensure I'm updating either the 10 and 11 docs, or just one, in a way that's appropriate per case.

I will make sure to stage my changes in a separate PR and merge that PR into this one.

Commit:
Change the location of the Navigation settings after [CTRL-3192]

quinntracy committed:
Merge pull request #9371 from mendix/qt-integrationfixes

Add Integration header fixes to Mx10

Yiyun333 committed:
Merge branch 'Mendix-11' into yl-continue-removing-mx-10-instances

Yiyun333 committed:
Update "Installing Mendix Studio Pro" again for 11

Yiyun333 committed:
Remove instances of "Studio Pro 10"

Commit:
Update preferences-dialog.md

Remvoed German. We don't support it anymore

WimJongeneel committed:
Update table with date format tokens

MariaShaposhnikova committed:
Merge branch 'development' into Mendix-11

WimJongeneel committed:
Fix example usage

Commit:
Add new help menu option (Contact Support)

Yiyun333 committed:
Merge branch 'Mendix-11' into yl-continue-removing-mx-10-instances

Yiyun333 committed:
Update External Database Connector for the Studio Pro 11 Refguide

Yiyun333 committed:
Remove space

sbont committed:
Added openAPI Documentation pane to Published REST service

Yiyun333 committed:
Update the "Access Rules" doc for Studio Pro 11 refguide

Yiyun333 committed:
Merge branch 'yl-continue-removing-mx-10-instances' of https://github.com/mendix/docs into yl-continue-removing-mx-10-instances

Yiyun333 committed:
Remove instances of "Mendix 10"

Yiyun333 committed:
Remove one instance of "Mendix version 10"

atennapel-mx committed:
Add new flag to mx create-module-package

Yiyun333 committed:
Remove the Access Rule Normalization section from Access Rules

Yiyun333 committed:
Remove todo item

mikita-haurykau committed:
Rename port fix to Cherry-pick

# Conflicts:
#	content/en/docs/refguide/modeling/menus/version-control-menu/merge-dialog.md
#	static/attachments/refguide/modeling/menus/version-control-menu/merge-dialog/merge-dialog.png

MariaShaposhnikova committed:
[SOLP-3259] Update history-dialog Mx11

MariaShaposhnikova committed:
Review

quinntracy committed:
Revise

quinntracy committed:
Add Private Values Commands

Yiyun333 committed:
Remove the whole info box in Update external-database-connection.md

gmzakpinar committed:
add view entity document updates with maia feature

MariaShaposhnikova committed:
Update repository-size.md

gmzakpinar committed:
added third party use for maia for oql

MariaShaposhnikova committed:
Merge branch 'Mendix-11' into Maria-Go-to-option

MariaShaposhnikova committed:
Update go-to-option.md

ester-nl committed:
Add documentation and sample for the notifications api

ester-nl committed:
Add documentation and sample for the notifications api

Yiyun333 committed:
Rename attachment folder to logic recommender

raghuveer-singh-mendix committed:
Merge branch 'Mendix-11' into Mpvr2AutoConversion

Yiyun333 committed:
Update attachment links for logic recommender

Yiyun333 committed:
Rename MD files of "Maia for Domain Model" and "Maia for Pages"

Yiyun333 committed:
Rename attachment folders

Yiyun333 committed:
Duplicate changes in Maia for Pages

Yiyun333 committed:
Duplicate changes in Maia for Domain Model

Yiyun333 committed:
Duplicate changes in Mendix AI Assistance (Maia)

Yiyun333 committed:
Duplicate changes in Preferences

Yiyun333 committed:
Duplicate other doc changes

Yiyun333 committed:
Duplicate screenshot changes

Yiyun333 committed:
Merge pull request #9354 from mendix/yl-update-doc-links-for-refguide10-and-howto10

Rename refguide and howto docs to refguide10 and howto10

Yiyun333 committed:
Merge pull request #9328 from mendix/yl-continue-removing-mx-10-instances

Continue removing Mx 10  intances for Mendix 11 Beta 1

YogendraShelke committed:
Added documentation for network timeout settings

YogendraShelke committed:
Renamed index file label

Yiyun333 committed:
Fix broken link: /howto10/monitoring-troubleshooting/monitoring-mendix-using-jmx/

MarkvanMents committed:
Proofread

MarkvanMents committed:
Proofread

Yiyun333 committed:
Duplicate PR #9390 in Mendix-11 branch - Re-implemented missing keyboard functionality

Yiyun333 committed:
Review Maia for OQL

MariaShaposhnikova committed:
Merge branch 'development' into Mendix-11

quinntracy committed:
Revision

quinntracy committed:
Remove folder

quinntracy committed:
Fix image link

MariaShaposhnikova committed:
Add beta to metadata

Yiyun333 committed:
Duplicate Maia status changes for refguide 10 in Mx 11

MariaShaposhnikova committed:
Move Mx11 Docs up and remove Mx10 release calendar from landing page

Yiyun333 committed:
Merge pull request #9424 from mendix/yl-duplicate-maia-status

Duplicate Maia status changes for refguide 10 in Mx 11

Yiyun333 committed:
Remove newly introduced Mx 10 instances in Mx 11 refguide

Yiyun333 committed:
Remove one new instance of Mendix version 10

raghuveer-singh-mendix committed:
Update repository-size.md

Yiyun333 committed:
Remove todo items that were answered or resolved

MariaShaposhnikova committed:
Review

MarkvanMents committed:
Rename Static Refguide Snippets to refguide10

MarkvanMents committed:
Restore refguide snippets

MariaShaposhnikova committed:
Add Beta warnings

quinntracy committed:
Mx11 updates for command-line tool

quinntracy committed:
Merge pull request #9438 from mendix/qt-fixes

Mx11 updates for command-line tool

Yiyun333 committed:
Remove the experimental label for Maia for OQL

Yiyun333 committed:
Merge branch 'Mendix-11' into pr/atmosfer/9401

ConnorLand committed:
Upload migration guide draft

ConnorLand committed:
Delete line

Yiyun333 committed:
Merge branch 'Mendix-11' into yl-duplicate-maia-renaming-changes

Yiyun333 committed:
Remove experimental labels

Yiyun333 committed:
Update preferences-dialog.md

Yiyun333 committed:
Review

Yiyun333 committed:
Merge branch 'Mendix-11' into pr/WimJongeneel/9379

Yiyun333 committed:
Add month examples

Yiyun333 committed:
Adjust the alert box order

quinntracy committed:
Revise

Yiyun333 committed:
Cross-reference review

ConnorLand committed:
Duplicate changes

ConnorLand committed:
Duplicate changes

All changes above Getting Started with Mobile duplicated

ConnorLand committed:
Duplicate changes

Everything above Using Mobile Capabilities done

ConnorLand committed:
Complete last of refguide 10

quinntracy committed:
Update via Slack

quinntracy committed:
Rename file

quinntracy committed:
Restore history

quinntracy committed:
Rename to SP 9

quinntracy committed:
Rename to 11

quinntracy committed:
Restore 10

quinntracy committed:
Rename indexes

MariaShaposhnikova committed:
Section on moving away from Team Server

MariaShaposhnikova committed:
Merge pull request #9451 from mendix/Maria-moving-from-team-server

Added section on moving away from Team Server

ester-nl committed:
Rename notifications-api.md to notification-api.md (singular)

ester-nl committed:
Add dialog-api

ester-nl committed:
Add preference-api

ester-nl committed:
Add to index

MariaShaposhnikova committed:
Update links to refguide10

MariaShaposhnikova committed:
Merge branch 'development' into Mendix-11

MariaShaposhnikova committed:
Fix links

MarkvanMents committed:
Fix broken link

quinntracy committed:
Fix broken link

quinntracy committed:
Fix screenshots

MariaShaposhnikova committed:
Merge pull request #9309 from WimJongeneel/ufc/cli-export-security-overview

Add docs for security overview export command

quinntracy committed:
Fix screenshots

quinntracy committed:
Merge pull request #9387 from atennapel/run/4642-mx-create-module-package-dep-sync

[RUN-4642] Add new flag to mx create-module-package docs

quinntracy committed:
Merge pull request #9378 from mauritselzinga-mx/patch-30

Update preferences-dialog.md

MariaShaposhnikova committed:
Add troubleshooting section on Git name/email for Mx11

MariaShaposhnikova committed:
Merge pull request #9457 from mendix/Maria-unknown-author

Add troubleshooting section on Git name/email for Mx11

MariaShaposhnikova committed:
Merge pull request #9280 from mendix/Maria-Go-to-option

Add go-to image to refguide10

quinntracy committed:
Revise

quinntracy committed:
Merge pull request #9401 from atmosfer/DUX-478_view_entity_document_changes

[DUX-478] Add View Entity Document Changes to Existing Documentation

quinntracy committed:
Include remote key-mapping out of sync side effect

MarkvanMents committed:
Remove old Mendix 10 references

MarkvanMents committed:
Merge remote-tracking branch 'origin/Mendix-11' into MvM-Mendix11Updates

MarkvanMents committed:
Add link to Mendix 11 Runtime API.

ConnorLand committed:
Fix broken links

ConnorLand committed:
Edit out minor version mentions

ConnorLand committed:
Introduce subsections

quinntracy committed:
Add anchor

quinntracy committed:
Update link

ConnorLand committed:
Add links

ConnorLand committed:
Update upgrading-from-10-to-11.md

quinntracy committed:
Merge pull request #9385 from sbont/rest-openapi-pane

Add OpenAPI Documentation pane to Published REST service

quinntracy committed:
Merge pull request #9459 from mendix/qt-addremotekeymapping

Include remote key-mapping out of sync side effect

ConnorLand committed:
improve breaking changes

ConnorLand committed:
typo

ConnorLand committed:
edit

ConnorLand committed:
Merge pull request #9439 from mendix/ctl-migration-guide

Mendix 11 Migration Guide

MariaShaposhnikova committed:
Remove a todo

MariaShaposhnikova committed:
Merge branch 'Mendix-11' of https://github.com/mendix/docs into Mendix-11

ConnorLand committed:
Fix link

ConnorLand committed:
Delete AR doc

ConnorLand committed:
Fix links

MariaShaposhnikova committed:
Review

quinntracy committed:
Fix info box

quinntracy committed:
Revise

quinntracy committed:
Merge pull request #9468 from mendix/qt-infobox

Fix info box

MariaShaposhnikova committed:
Merge branch 'development' into Mendix-11

quinntracy committed:
Merge branch 'development' into Mendix-11

MariaShaposhnikova committed:
Merge branch 'development' into Mendix-11

MariaShaposhnikova committed:
Update Security overview for Mx 11 and Mx10

Yiyun333 committed:
Duplicate #9434 Java version migration

Yiyun333 committed:
Merge pull request #9469 from mendix/yl-duplicate-recent-dev-merges

Duplicate #9434 Java version migration

MariaShaposhnikova committed:
Add info on merge conflicts to Mx 10

MariaShaposhnikova committed:
Merge branch 'Mendix-11' of https://github.com/mendix/docs into Mendix-11

ConnorLand committed:
Add warning and remove todo

exec-astraea committed:
Mentioned mx.logger removal and changes around empty values and empty strings

ConnorLand committed:
Update prerequisites.md

ConnorLand committed:
Delete todos

Both dojo docs need to stay up, and the mobile todo is resolved in #9471

Yiyun333 committed:
Update Maia screenshots

MarkvanMents committed:
Update Database Source information.

Yiyun333 committed:
Merge branch 'Mendix-11' into yl-duplicate-maia-renaming-changes

Yiyun333 committed:
Screenshot fixes

ester-nl committed:
add missing `this` for the menuid and remove `assets` from manifest in the dialog api file

MarkvanMents committed:
Replace login2 mentions with the Mendix 11 Client API

MarkvanMents committed:
Proofread

ConnorLand committed:
Increase Atlas 3 to 4 migration guidance

ConnorLand committed:
edit

ConnorLand committed:
Merge pull request #9477 from mendix/ctl-add-atlas-migration

Increase Atlas 3 to 4 migration guidance

ConnorLand committed:
Update upgrading-from-10-to-11.md

ConnorLand committed:
Merge pull request #9472 from exec-astraea/client-side-breaking-changes

Client-side breaking changes

mikita-haurykau committed:
SOLP-3302 removing git-merge command from refguide

MarkvanMents committed:
Respond to review

MarkvanMents committed:
Respond to review and create new section for working with authentication tokens

MarkvanMents committed:
Respond to review comments.

MarkvanMents committed:
Respond to review.

MarkvanMents committed:
Merge branch 'appext/1041-notifications-api-documentation' of https://github.com/ester-nl/docs into pr/ester-nl/9403

quinntracy committed:
Update naming for SP 9 folder

quinntracy committed:
Update SP 10 naming

quinntracy committed:
SP 11 naming updates

ester-nl committed:
Merge branch 'mendix:development' into appext/1041-notifications-api-documentation

quinntracy committed:
More SP 9 renamiing

MarkvanMents committed:
Merge pull request #9460 from mendix/MvM-Mendix11Updates

Remove old Mendix 10 references

MarkvanMents committed:
Merge pull request #9319 from lvanengelen/patch-2

[RUN-3523] Remove ObjectManagement.StrictChangeBehavior runtime setting

quinntracy committed:
More renaming

quinntracy committed:
More renaming

MarkvanMents committed:
Rename for splitting APIs

MarkvanMents committed:
Merge branch 'Mendix-11' into qt-apistructure

MarkvanMents committed:
URLs for extensibility API restructure

MarkvanMents committed:
Update links for Extensions versioning

MarkvanMents committed:
Merge branch 'qt-apistructure' into pr/ester-nl/9403

MarkvanMents committed:
correct bad links

ester-nl committed:
Change webextensions folder name to extensions for v11

quinntracy committed:
Remove bullet

quinntracy committed:
Review

quinntracy committed:
Merge branch 'appext/1041-notifications-api-documentation' of https://github.com/ester-nl/docs into pr/9403

MariaShaposhnikova committed:
Merge branch 'development' into Mendix-11

Yiyun333 committed:
Merge pull request #9404 from mendix/yl-duplicate-maia-renaming-changes

Duplicate Rename Maia sessions PR #9273 in Mendix-11 branch

Yiyun333 committed:
Merge pull request #9422 from mendix/yl-duplicate-keyboard-changes

Duplicate PR #9390 in Mendix-11 branch - Re-implemented missing keyboard functionality

quinntracy committed:
Move changes to refguide10

quinntracy committed:
Merge branch 'Mendix-11' into qt-privatevaluescommand

quinntracy committed:
Merge pull request #9400 from mendix/qt-privatevaluescommand

Add show and delete private values Mx11

quinntracy committed:
Move changes to refguide10

quinntracy committed:
Remove section

quinntracy committed:
Merge pull request #9384 from envialon/ctrl/3192-Back-And-Forward-Buttons-By-Default-to-mx-11

Change the location of the Navigation settings after [CTRL-3192]

Yiyun333 committed:
Adjust Maia docs for 11 beta 1 and remove Mx 10 instances

Yiyun333 committed:
Merge pull request #9493 from mendix/yl-remove-mx10-info-from-11-beta-1

Adjust Maia docs for 11 beta 1 and remove Mx 10 instances

quinntracy committed:
Move update to refguide10

MariaShaposhnikova committed:
Duplicate changes to Mx10 on system requirements from PR 9495

quinntracy committed:
Merge branch 'Mendix-11' into ctrl/3190-contact-support-option-mx-11

quinntracy committed:
Merge pull request #9382 from envialon/ctrl/3190-contact-support-option-mx-11

Add new help menu option (Contact Support)

Yiyun333 committed:
Remove Mx 10 instances from the Installing Mendix Studio Pro Offline section

Yiyun333 committed:
Merge branch 'Mendix-11' of https://github.com/mendix/docs into Mendix-11

MariaShaposhnikova committed:
Merge branch 'Mendix-11' into Maria-update-history-dialog-11

MariaShaposhnikova committed:
Transfer changes to Mx10

MariaShaposhnikova committed:
Merge pull request #9397 from mendix/Maria-update-history-dialog-11

[SOLP-3259] Update history-dialog Mx 11

MariaShaposhnikova committed:
Merge pull request #9392 from mikita-haurykau/solp/2255-11-rename-port-fix

[SOLP-2255] Mendix-11 Rename port fix to Cherry-pick

Yiyun333 committed:
Merge pull request #9379 from WimJongeneel/ufc/update-date-formatting-mendix-11

Update table with date format tokens (Mx 11)

MariaShaposhnikova committed:
Merge branch 'Mendix-11' into pr/raghuveer-singh-mendix/9295

quinntracy committed:
Merge pull request #9278 from sergey-me/ctrl/3213-Release-the-Microsoft-Defender-whitelist-feature-to-11

[CTRL-3213] Release the Microsoft Defender whitelist feature to v11

MariaShaposhnikova committed:
Align changes for Mx 11 and Mx 10

MariaShaposhnikova committed:
Merge pull request #9295 from raghuveer-singh-mendix/Mpvr2AutoConversion

Updated documentation for Auto Upgrade of application to MPRv2.

ConnorLand committed:
Merge pull request #9471 from mendix/ctl-resolve-todos

Resolve Mx11 todo with warning

MariaShaposhnikova committed:
Merge branch 'Mendix-11' into pr/mikita-haurykau/9479

MariaShaposhnikova committed:
Remove git-merge from Mx 10

MariaShaposhnikova committed:
Delete info on git merge

ConnorLand committed:
Merge pull request #9305 from MendixBart/pe/PAG-2100-remove-building-block-documentation-URL

Remove mention of Documentation URL for Building Block

ConnorLand committed:
Update network-request-timeout.md

ConnorLand committed:
Edit

ConnorLand committed:
Merge pull request #9407 from YogendraShelke/Mendix-11

Added documentation for network timeout settings

ConnorLand committed:
remove mx10 text from mx11 doc

ConnorLand committed:
Update best-practices.md

ConnorLand committed:
Merge pull request #9349 from mendix/ctl-update-inheritance

Upload inheritance support change

ConnorLand committed:
Merge pull request #9445 from mendix/ctl-appcenter-10-dupes

Duplicate changes

ConnorLand committed:
Merge pull request #9331 from mendix/ctl-sync-error-fix

Correct System.SynchronizationFailure to System.SynchronizationError

ConnorLand committed:
Merge branch 'Mendix-11' into ctl-remove-native-AR

ConnorLand committed:
Merge pull request #9464 from mendix/ctl-remove-native-AR

Delete deprecated Augmented Reality doc

quinntracy committed:
Notification proof

quinntracy committed:
Dialog proof

quinntracy committed:
Preferences proof

MariaShaposhnikova committed:
Merge branch 'Mendix-11' into pr/mikita-haurykau/9479

MariaShaposhnikova committed:
Merge pull request #9479 from mikita-haurykau/solp/3032-MX11-remove-MPRv2-merge-driver

SOLP-3302 removing git-merge command from refguide

MariaShaposhnikova committed:
Duplicate information for Mx 10 from 9494 PR

MariaShaposhnikova committed:
Merge branch 'Mendix-11' of https://github.com/mendix/docs into Mendix-11

MariaShaposhnikova committed:
Remove todo comments

MariaShaposhnikova committed:
Fix a link

ConnorLand committed:
Fix link error

This bullet refers to a doc that was intentionally removed. This bullet should not have remained, so now all is as it should be.

quinntracy committed:
Final proof

quinntracy committed:
Merge pull request #9403 from ester-nl/appext/1041-notifications-api-documentation

Add documentation and sample for the notifications api

quinntracy committed:
Merge branch 'Mendix-11' into qt-apistructure

quinntracy committed:
Merge pull request #9449 from mendix/qt-apistructure

Update Extensibility API structure

quinntracy committed:
Fix links

MariaShaposhnikova committed:
Merge branch 'development' into Mendix-11

MariaShaposhnikova committed:
Update latest releases
